nodebuilder/rpc/opts.go

```diff
@@ -0,0 +1,19 @@
+package rpc
+
+import (
+	"github.com/celestiaorg/celestia-node/service/rpc"
+)
+
+// SetRPCPort configures Node to expose the given port for RPC
+// queries.
+// TODO(distractedm1nd): Once the config migrates to this package, define as a method
+func SetRPCPort(cfg *rpc.Config, port string) {
```

Comment:
refer to comments regarding setters earlier

Reply:
yeah, in this case specifically it might be better to just leave out the setters since they have different syntax. Depends on main discussion thread I'd say